src/apps/frontend/webpack.base.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.md$/,
```

Comment:
Can you add please share why this change is needed? My knowledge of webpack is bit dated, rusty at the moment.

Reply:
This is added to load the readme files in earlier implementation, I have removed it now.